membership-common/src/main/scala/com/gu/memsub/subsv2/services/SubscriptionService.scala

```diff
@@ -98,30 +84,17 @@ class SubscriptionService[M[_]: Monad](pids: ProductIds, futureCatalog: => M[Cat
       else
         s"subscriptions/${name.get}" // FIXME: equivalent to ?charge-detail=last-segment which returns even removed historical charges. We should not have this as default.
 
-    for {
-      subJson <- rest.get[JsValue](url)(idReads, logPrefix)
-      catalog <- futureCatalog
-    } yield for {
-      jsValue <- subJson.withLogging(s"get subscription $name from zuora").toOption
-      subscription <- SubscriptionTransform
-        .getSubscription(catalog, pids)(jsValue)
-        .withLogging(s"getAllValidSubscriptionsFromJson for $name")
-        .toOption
-    } yield subscription
```

Comment:
we used to have a bit of a rigmarole getting the zuora response as a JsValue, and then running it through the deserliasers (together with the catalog) to get it all the way to the high level model.
This is all gone and we just deserialise the response straight into the (low level) model